addons/transifex/static/src/views/reload_code_translations_hook.js

```diff
@@ -0,0 +1,15 @@
+/** @odoo-module **/
+
+import { useService } from "@web/core/utils/hooks";
+
+const { useComponent } = owl;
+
+export function useReloadCodeTranslationsButton() {
+    const component = useComponent();
+    const orm = useService("orm");
+
+    component.onClickReloadCodeTranslations = async () => {
+        await orm.call("transifex.code.translation", "reload", [], {});
+        window.location.reload();
```

Comment:
```suggestion
        browser.location.reload();
```
browser comes from @web/core/browser/browser. Will be the same in prod, but it's testable (mocked in tests)